scripts/generate-notices.py

```
@@ -46,45 +60,142 @@ def tag(prefix: str) -> str:
+        # --ignore matches by PREFIX, so it must be given full stdlib package
+        # paths. Reducing them to first path segments yields the bare token
+        # "go" (from go/ast, go/build), which then also matches every module
+        # under go.opentelemetry.io, go.uber.org, golang.org/x, google.golang.org
+        # and gopkg.in, silently dropping every one of them from the notices
+        # with no error raised. The set is recomputed per platform because
+        # platform-specific stdlib packages, such as internal/syscall/windows,
+        # exist only under their own GOOS.
+        stdlib = go_out(["list", "std"], goos, goarch)
+        ignore = ",".join([*sorted(line for line in stdlib.splitlines() if line), local])
+        env = go_env(goos, goarch)
+        subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)